VideoWeb/VideoWeb/Extensions/ConfigureAuthSchemeExtensions.cs

```diff
@@ -117,6 +117,7 @@ private static void AddPolicies(AuthorizationOptions options, IList<IProviderSch
             {
                 [AppRoles.JudgeRole] = new[] { AppRoles.JudgeRole },
                 [AppRoles.VhOfficerRole] = new[] { AppRoles.VhOfficerRole },
+                ["Host"] = new[] {AppRoles.JudgeRole, AppRoles.StaffMember },
```

Comment:
change this to use a constant after updating AppRoles

Reply:
if we add constants, Authorize is not working for to pick multiple roles